### PR TITLE
Update JuliaExperimental for Julia 1.0 and latest JuliaInterface

### PR DESCRIPTION
**Pull request by fingolfin**
Note that the test still produce diffs and even crash for me, I'll look into that next.

### CODE REVIEW

File: JuliaExperimental/tst/testall.g [hunk 1]
```diff
@@ -1,35 +1,14 @@
 #
-# JuliaExperimental: Experimental code for the GAP Julia integration
+# JuliaInterface: Test interface to julia
```
Comment (ThomasBreuer) [inline JuliaExperimental/tst/testall.g]: really?

File: JuliaExperimental/julia/zlattice.jl [hunk 2]
```diff
@@ -9,13 +9,15 @@ module GAPZLattice
...
+import LinearAlgebra: diag
+
+import Base: abs, convert, copy, deepcopy, haskey, inv, lcm, length,
+             map, push!, sign, size, sum, trunc, zero, zeros
 
 export LLLReducedGramMat, ShortestVectors, OrthogonalEmbeddings
 
 
-doc"""
+raw"""
```
Comment (ThomasBreuer) [inline JuliaExperimental/julia/zlattice.jl]: raw?

File: JuliaExperimental/tst/testall.g (hunk 1 above)
Comment (fingolfin) [reply JuliaExperimental/tst/testall.g]: Ouch

File: JuliaExperimental/julia/zlattice.jl (hunk 2 above)
Comment (fingolfin) [reply JuliaExperimental/julia/zlattice.jl]: The problem is the `$` later in the string, which Julia tries to interpolate, like in a shell script. We could remove that; but @sebasguts suggested this alternative change, which disables variable interpolation for the string.

Comment (sebasguts) [reply JuliaExperimental/julia/zlattice.jl]: We could probably also do `\$`, but I would try to investigate into this when we actually start to build julia documentation.

**Pull request merged**